sidebars.js

```diff
@@ -104,7 +104,8 @@ const sidebars = {
             items: [
               'guides/integration-prebid',
               'guides/publisher-client-side',
-              'guides/integration-prebid-advanced',
+              'guides/integration-prebid-client-side',
+              'guides/integration-prebid-server-side',
```

Comment:
i think the server/client-side guides should be child pages under guides/integration-prebid ?

Reply:
@sunnywu definitely do not want to add another level on the sidebar. Plus we don't have styling for it. Not sure what the best approach is for this.

Reply:
For under `Web` section, maybe we don't show the Client/Server side Integration sub pages? If we lead people to start reading from prebid.js Integration Overview, i don't think we neeed to list Client-Side/Server-Side pages Under `Web`?

It's still listed under `Prebid `subsection anyway

Reply:
@sunnywu I hear you. Currently, if I take it off the sidebar it messes with the way the page is rendered.
I have a plan to make audience-specific sidebars when I can get to it. That will give us some leeway because we won't need to have the Publisher Integrations level. I believe that's the best solution for organization. Will discuss with KK also but currently I feel the best plan is to include all. LP might have a better suggestion. Also FYI KK wanted Prebid listed in both places (in both the summary file and the sidebar).

Reply:
We could set up styling for a 3rd level.
We're really set up for having everything linked from the sidebar - the whole nav system is designed around the idea of a "current page" that you're on which is highlighted in the sidebar. Changing that would probably require some major design re-thinking and I don't think it's worth it.

Reply:
@lionell-pack-ttd understood. Having audience-specific menu options was actually an original requirement but at the time I didn't think we could even do it. Now, we could and it's on my list to get to quite soon. Need KK input here also.

Reply:
FYI discussed with KK today 12/20 and he was in agreement on doing audience-specific sidebars.